### PR TITLE
goblin and while for odd and even

### PR DESCRIPTION
**Pull request by poornima-ezhilmathi**
Leo, i have added the while loop for odd and even as a method in the same scripts alone with goblin program. please refer.
Thank you for your time.

### CODE REVIEW

File: goblin_hunt.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+def goblin_hunt():
+    print("Welcome to the Angry Goblin Hunt")
+    print("n award-winning game full of adventure and excitement (!)")
+    input("Type in your name:")
+    print("Stephen, do you think you can find the goblin hiding in the kitchen cupboards?")
+    print("|_||_||_||_||_|")
+    i=0
+    while i<=5:
```
Comment (leocyriac) [inline goblin_hunt.py]: User is getting only 6 chances. Is that a right thing to do?

File: goblin_hunt.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+def goblin_hunt():
+    print("Welcome to the Angry Goblin Hunt")
+    print("n award-winning game full of adventure and excitement (!)")
+    input("Type in your name:")
+    print("Stephen, do you think you can find the goblin hiding in the kitchen cupboards?")
```
Comment (leocyriac) [inline goblin_hunt.py:5]: This needs to be correctd.

image

**Review by leocyriac — CHANGES_REQUESTED** [review]
Apart from the inline comments. Please separate out the odd even programs by the type of loop. It doesn't make sense to have the same program printing the same set of numbers twice. This git repository should be something for you to refer back or show to somebody as a programming portfolio.

File: goblin_hunt.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+def goblin_hunt():
+    print("Welcome to the Angry Goblin Hunt")
+    print("An award-winning game full of adventure and excitement (!)")
+    input("Type in your name:")
+    print("Stephen, do you think you can find the goblin hiding in the kitchen cupboards?")
+    print("|_||_||_||_||_|")
+    while True:
+          val=int(input("Which cupboard do you think the goblin is in [type in number]:"))
+          if val==3:
+            False
```
Comment (leocyriac) [inline goblin_hunt.py]: what's this doing here?

**Review by leocyriac — APPROVED** [review]
Looks good. Don't merge Goblin.. we will modify it in our next class

File: goblin_hunt.py (hunk 3 above)
Comment (poornima-ezhilmathi) [reply goblin_hunt.py]: i removed false, this is not necessary

**Pull request closed without merging**